utils/constants/graph-constants.js

```diff
@@ -0,0 +1,83 @@
+export const graphOptions = {
+  plugins: {
+    legend: {
+      display: false, // Set to false to hide the legend
+    },
+  },
+  responsive: true,
+  maintainAspectRatio: false, // Allow dynamic aspect ratio
+  scales: {
+    y: {
+      display: false, // Hide y-axis labels
+    },
+    x: {
+      display: false, // Hide x-axis labels
+    },
+  },
+};
+
+export const graphConfig = [
```

Comment:
Same suggestion here, use `GRAPH_CONFIG` instead of `graphConfig`. Apart from that, everything looks fine.

Reply:
Done